src/StakingNode.sol

```diff
@@ -237,20 +223,27 @@ contract StakingNode is IStakingNode, StakingNodeEvents, ReentrancyGuardUpgradea
             // (32ETH in the absence of slasing, and less than that if slashed)
             uint256 effectiveBalanceGwei = validatorFields[i].getEffectiveBalanceGwei();
 
-            emit VerifyWithdrawalCredentialsCompleted(validatorIndices[i], oracleTimestamp, effectiveBalanceGwei);
+            emit VerifyWithdrawalCredentialsCompleted(validatorIndices[i], beaconTimestamp, effectiveBalanceGwei);
             
-            if (effectiveBalanceGwei > 0) {
-                // If the effectiveBalanceGwei is not 0, then the full stake of the validator
-                // is verified as part of this process and shares are credited to this StakingNode instance.
-                // This assumes StakingNodesManager.sol always stakes the full 32 ETH in one go.
-                // effectiveBalanceGwei *may* be less than DEFAULT_VALIDATOR_STAKE if the validator was slashed.
-                unverifiedStakedETH -= DEFAULT_VALIDATOR_STAKE;
-
-                emit ValidatorRestaked(validatorIndices[i], oracleTimestamp, effectiveBalanceGwei);
-            }
+            // If the effectiveBalanceGwei is not 0, then the full stake of the validator
```

Comment:
we should change this comment to talk about how verifyWithdrawalCredentials can only be called for non-exited validators therefore effectiveBalanceGwei >0 always

Reply:
added a note for this